docs/bin/README.md

```diff
@@ -129,13 +188,13 @@ I don't know the formula for calculating the correct number of threads... this w
 ### Usage
 
 ```
-usage: gopro-dashboard.py [-h] [--font FONT] [--gpx GPX] [--privacy PRIVACY]
+usage: gopro-dashboard.py [-h] [--font FONT] [--gpx GPX] [--video-time-start {mtime,atime,ctime}] [--video-time-end {mtime,atime,ctime}] [--privacy PRIVACY]
```

Comment:
this could be --video-time {start-mtime, start-atime, start-ctime, end-mtime, end-atime, end-ctime} for simplicity
Later we can add more rules to get the video time from mp4 metadata for different camera vendors